src/Math-Matrix/PMMatrix.class.st

```diff
@@ -588,6 +588,17 @@ PMMatrix >> orthogonalize [
 
 ]
 
+{ #category : #'as yet unclassified' }
+PMMatrix >> principalDiagonal [
+	"https://en.wikipedia.org/wiki/Diagonal#Matrices for definitions"
+	| diag |
+	"Check for square"
```

Comment:
I would say that the comment is redundant as the code is intention revealing enough.